posts/bootcamp.md

```diff
@@ -0,0 +1,72 @@
+# Bootcamp
+
+Hi everyone!
+
+I'm really excited to let you know that from January 2025, we're going to be running a **[Learn to Code Bootcamp](https://bootcamp.exercism.org)**!
+
+## Some background
+
+I've been wanting to help total beginners for a long time. 
+As I mentioned in the "[2M users but no money in the bank](https://exercism.org/blog/september-2024-restructure)" blog post in September,
+over 95% of people who try to learn to code give up, which I find unacceptable.
+
+We're building a new platform that will sit alongside Exercism which is going to be dedicated to beginners, which we'll be launching in mid-2025. 
+We've already put a lot of effort into this and I'm really proud of how it's evolving.
+I'll be talking a lot more about it in the coming months
+
+But before launching that, we're going to run a 6 month bootcamp, that will use some of the technology we're building. This will allow me to get super-close to students' learning experience, to refine and develop the platform as real people learn from it, and help me understand where students get stuck, and work out how to help them further.
+
+## Some details
+
+The bootcamp will be in two parts:
+1. Learn to code
+2. Learn front-end web dev
+
+Students can take just the first part (3 months from Jan-March) or the whole course (6 months from Jan-July). 
+I may also just offer the second part later, for existing developers who want to learn some web development skills.
+
+The bootcamp is remote, part-time, and priced as cheaply as possible with 
+pricing parity available, along with discounts for students and those who are currently unemployed
+
+The whole focus is on learning through doing. 
+Like Exercism, people will learn by coding lots.
+We're going to start with smaller exercises and evolve into bigger projects.
+This is how I learnt 3 decades ago, and I strongly feel that learning by doing is the best way for a brain to really build an intuitive and natural understanding of how to do something.
+
+Rather than me regurgitating everything, it's probably best to scroll through the **[Bootcamp website](https://bootcamp.exercism.org)** for all the details!
+
+There are two ways that you can help!
+
+## Help by spreading the word
+
+The biggest thing you can do to help is to encourage everyone you know to come and join!
+
+I've written many times about how valuable I think learning to code is for pretty much everyone today.
+Even if you don't want a career in tech, coding a great skill to have. It nurtures critical thinking skills, helps you understand how so much of the world worlds, and with the power of LLMs, its increasingly easy for people to piece together different little bits of code to do useful things.
+
+We're setting up an affiliate scheme where your friends or followers can get a 20% discount, and you get a 20% "commission" on the sale (which you can choose not to take to support Exercism if you prefer!)
```

Comment:
```suggestion
We're setting up an affiliate scheme where your friends or followers can get a 20% discount, and you get a 20% "commission" on the sale (which you can choose not to take to support Exercism if you prefer!).
```

Reply:
The `!` in the bracket means the `.` is redundant.